packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -85,14 +85,14 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
   const { t, i18n } = useLocale();
   const utils = trpc.useUtils();
 
-  const { hasPaidPlan } = useHasPaidPlan();
-
   const { step, form, reload, setReload, teamId } = props;
   const { data: _verifiedNumbers } = trpc.viewer.workflows.getVerifiedNumbers.useQuery(
     { teamId },
     { enabled: !!teamId }
   );
 
+  const { hasActiveTeamPlan } = useHasActiveTeamPlan(teamId);
```

Comment:
@joeauyeung Now we only allow custom templates on team workflows but not on User workflows for user's that are part of a team or org